scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -195,6 +195,8 @@ public static BuildTargetResult PackagePkgProjects(BuildTargetContext c)
                 var fileName = Path.GetFileName(file);
                 File.Copy(file, Path.Combine(Dirs.Corehost, fileName), true);
                 runtimeCount += (fileName.StartsWith("runtime.") ? 1 : 0);
```

Comment:
Now that this has been expanded to do more, `&& fileName.Contains("DotNetHost")`